.github/src/context.js

```diff
@@ -60,9 +60,9 @@ export async function extractInputs(github, context, core) {
       // empty for non-fork PRs.
       issue_number = pull_requests[0].number;
     } else {
-      // For fork PRs, we must call an API in the head repository to get the PR number in the target repository
```

Comment:
Official GH terminology is "base" for the target of the PR, and "head" for the source of the PR.